server/schema.sql

```diff
@@ -30,4 +35,16 @@ CREATE TABLE examples (
     example_task             TEXT NOT NULL,
     example_action           TEXT NOT NULL,
     example_result           TEXT NOT NULL,
-);  */
\ No newline at end of file
+);  */
+
+INSERT INTO users (id, first_name, last_name, email, password, created_at) VALUES (1,'John', 'Smith', 'j.smith@johnsmith.org', 'prawn345!', '' );
```

Comment:
When we add data to our tables it is good practice to have these in 2 seperate files. 1 file for creating the tables `schema.sql` and another for inserting data into those tables (e.g. something like `data.sql`)